Added the wsdl resource file

randythomas opened the pull request:
Closes #418

The first part of the fix for Jira: https://training-feedback.redhat.com/browse/AD221-2

psolarvi reviewed (changes requested):
Please see the blocking comments in the content PR. 

I'm going to block this PR until the comments are resolved, and until we can we start the release of the fix, otherwise we might break the GE for the current students.

randythomas commented:
I removed WSDLRetrievalRoutBuilder.java and WSDLRetrievalRouteBuilderTest.java files from apps and solution.
I also removed the camel-http4 dependency from the two pom.xml files.
